src/components/CarListComponent/CarListComponent.tsx

```diff
@@ -0,0 +1,121 @@
+import { useEffect, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import {
+  Order_By,
+  Query_GetCarsQueryVariables,
+  useQuery_GetCarsQuery,
+} from '../../api/graphql/__generated__/graphql-types';
+import { isValidUuid } from '../../helpers/validators';
+import ErrorMessage from '../ErrorMessage/ErrorMessage';
+import FilterComponent from '../FilterComponent/FilterComponent';
+import HeaderList from '../HeaderList/HeaderList';
+import LoadingComponent from '../LoadingComponent/Loading';
+import BodyList from './BodyList/BodyList';
+import { Container } from './CarListComponentStyled';

```

Comment:
why is this called Container and the file is `CarListComponentStyled` it seems this is the only thing exported from the file, also I see you are using this container multiple times, isn't better to make this part of your layout?

Reply:
Reason: I'm using Container name trying to stick to styled components naming for divs

Solution: i will create a global "Container" style and call this "Container" in every style and rename components containers